Core1Platform/02_CoreCapabilities.md

```diff
@@ -0,0 +1,51 @@
+# Core capabilities
+
+The *Actindo Core1 Platform* provides you with the following:
+- [Standalone solution or modernization of legacy ERP systems](md#standalone-solution-or-modernization-of-legacy-erp-systems)
+- [Omni-channel integration](#omni-channel-orchestration)
+- [Data transformation and process orchestration](#data-transformation-and-process-orchestration)
+- [Monitoring and reporting](#monitoring-and-reporting)
+ 
+
+## Standalone solution or modernization of legacy ERP systems
+
+The *Actindo Core1 Platform* supports the business of companies in wholesale and retail, sale via marketplaces or social commerce. It treats online commerce the same as stationary trading. Each store is its own marketplace, each transaction at a pay desk an incoming payment or refund.  
```

Comment:
Instead of "It treats..." i would write something like "It enables Unified Commerce by treating digital and traditional stationary commerce in a comparable way". And then you can give the example that stationary store is treated like a webshop. And a webshop order like a sale process in the PoS-System
Wording "Unified Commerce" is imho important

Reply:
done